sdk/template/Azure.Template/src/Generated/TemplateClient.cs

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Core.Pipeline;
+
+namespace Azure.Template
+{
+    /// <summary> The Template service client. </summary>
+    public partial class TemplateClient
+    {
+        private static readonly string[] AuthorizationScopes = new string[] { "https://api.adu.microsoft.com/.default" };
+        private readonly TokenCredential _tokenCredential;
+        private readonly HttpPipeline _pipeline;
+        private readonly string _vaultBaseUrl;
```

Comment:
This should be a `Uri` type.

Reply:
If swagger does not define its format uri and no customization configuration added, codegen will generate it as string. This is an example, we should not add customization to do so.